app/importers/mahonia_mapper.rb

```diff
@@ -28,6 +29,16 @@ def fields
     end
   end
 
+  def embargo_release_date
+    return if metadata['embargo_date'].nil?
+    Array(Mahonia::EdtfLiteral.new(metadata['embargo_date']))
```

Comment:
I don't think we want `embargo_date` to be an `EdtfLiteral`, but probably just a `Date`? cf. https://github.com/curationexperts/mahonia/blob/master/spec/factories/etds.rb#L40

I think we probably also need a ticket to ensure the embargo is set in the `RecordImporter` if an embargo date is available. Or does the actor stack already do this and we just need to test?

Reply:
I'll check the actor stack about this, I suspect it does, and get back to you.